gitlab.go

```diff
@@ -508,6 +510,14 @@ func newResponse(r *http.Response) *Response {
 // populatePageValues parses the HTTP Link response headers and populates the
 // various pagination link values in the Response.
 func (r *Response) populatePageValues() {
+	if totalPages, ok := r.Response.Header["X-Total-Pages"]; ok && len(totalPages) > 0 {
+		r.TotalPages, _ = strconv.Atoi(totalPages[0])
```

Comment:
I would suggest to use `r.Response.Header.Get("X-Total-Pages")` instead... So something like this:

```
if totalPages := r.Response.Header.Get("X-Total-Pages"); totalPages != "" {
  r.TotalPages, _ = strconv.Atoi(totalPages[0])
}
```